docs/release/trg-7/trg-7-04.md

```diff
@@ -73,8 +73,8 @@ Make sure to also include test dependencies. For a maven-based java project you
       <plugin>
```

Comment:
I think this is misleading/wrong. The plugin section is included in `<build></build>`. Here it seems like the plugin section is on the same level as `<pluginRepositories>`. I think this should be changed with this PR as well

Reply:
like this?
![image](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/121097161/01466dfb-bdad-44c6-8792-0952e85f6be9)